pandapower/build_bus.py

```diff
@@ -329,9 +329,23 @@ def _fill_auxiliary_buses(net, ppc, bus_lookup, element, bus_column, aux):
     element_bus_idx = bus_lookup[net[element][bus_column].values]
     aux_idx = bus_lookup[aux[element]]
     ppc["bus"][aux_idx, BASE_KV] = ppc["bus"][element_bus_idx, BASE_KV]
-    if net._options["mode"] == "opf":
+    if hasattr(net, '_options'):
+        if net._options["mode"] == "opf" and element is "xward":

```

Comment:
Why are you using "is xward" for comparison here?

Reply:
This is not needed, will change this before commiting.

Reply:
to compare strings please use "==" instead of "is"

(see e.g. https://stackoverflow.com/questions/1504717/why-does-comparing-strings-using-either-or-is-sometimes-produce-a-differe)